### ISSUE TITLE
React Player fetches low resolution thumbnail in light mode

### ISSUE THREAD

**Issue by caedney** [cookpete/react-player#1448]
#### Current Behavior
When light mode is enabled the video thumbnail being fetched is a low resolution image even though a high resolution image is available.

#### Expected Behavior

When a high resolution thumbnail is available, light mode should use this as the poster image, rather than having to manually send through a separate image url.

#### Environment
- Browser: Chrome
- Operating system: macOS Monterey
- CodeSandbox example: https://codesandbox.io/s/light-player-1qgn4m

#### Other Information

In the code example, when we disable light mode we can see the high resolution thumbnail available.

**Comment by cookpete** [comment]
Duplicate of https://github.com/cookpete/react-player/issues/720

**Comment by caedney** [comment]
@cookpete You mention as a possible solution to pass through an image url to the light prop, but if our client has updated all their YouTube videos with high resolution images, adding these again is double the work. Is there no way to attempt fetching the high resolution thumbnail and have the low resolution image as a fallback?

**Comment by cookpete** [comment]
Yep, I’m not against fixing this (hence why I didn’t close this issue). Just pointing out a similar discussion from the past.
